libraries/AP_Logger/LogStructure.h

```diff
@@ -571,6 +571,24 @@ struct PACKED log_NKF5 {
     float posErr;
 };
 
+struct PACKED log_XKF5 {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;
+    uint8_t normInnov;
+    int16_t FIX;
+    int16_t FIY;
+    int16_t AFI;
+    int16_t HAGL;
+    int16_t offset;
+    int16_t RI;
+    uint16_t meaRng;
+    uint16_t errHAGL;
+    float angErr;
+    float velErr;
+    float posErr;
+    float RErr[3]; // set according to defined maximum allowed EKF3 cores
```

Comment:
Intead of the new structure I think this should go into XKF3. Sorry for suggesting XKF5 on discord. We can do it as a single float on XKF3 as that msg is per-lane. 
Note that while the XKF3 structure is shared with EKF2, I think that is OK, as we likely will bring across your new lane switching code to EKF2 as well

Reply:
Changed. In the meantime for EKF2, since we are using the same NKF3 structure, setting the error value to be 0. Hope that’s fine for now.